Source/MediaInfo/MediaInfo_Config_MediaInfo.cpp

```diff
@@ -1788,8 +1789,7 @@ Ztring MediaInfo_Config_MediaInfo::File_Duplicate_Set (const Ztring &Value_In)
     }
     if (!ToReturn.empty())
         ToReturn.erase(ToReturn.begin()); //Remove first ";"
-
-    CS.Leave();
+    }

```

Comment:
ok. don't forget the compilation error due to this change. (AppVeyor is complaining)